packages/massa-web3/src/experimental/crypto/interfaces/hasher.ts

```diff
@@ -0,0 +1,3 @@
+export default interface Hasher {
```

Comment:
Whats the benefits of using interfaces here ?? i have the feeling that it adds complexity but i don't see why its needed

Reply:
maybe you just wanted to use "types" ?

Reply:
All the benefits of dependency injections: make test and evolution easier.